src/main.rs

```diff
@@ -46,9 +45,11 @@ struct Args {
     #[structopt(long = "features", value_name = "FEATURES")]
     /// Space-separated list of features to activate
     features: Option<String>,
-    #[structopt(long = "all-features")]
-    /// Activate all available features
-    all_features: bool,
+    // XXX: Cargo 0.40 requires that if the `specs` (in `resolve_with_previous`) is empty,
+    //      `all_features` must be true. Hence the disable here.
+    // #[structopt(long = "all-features")]
+    // /// Activate all available features
+    // all_features: bool,
```

Comment:
I need to remove this flag.

Reply:
Why? It's definitely a thing cargo supports.

Reply:
Otherwise, when we pass the non true `opts.all_features`, it will panic as in https://github.com/sfackler/cargo-tree/pull/85#issuecomment-561706918.

Reply:
That means that the implementation needs to change to avoid that panic, not that we're just going to remove `--all-features`.